lib/util/git/config.spec.ts

```diff
@@ -1,9 +1,31 @@
+import { mocked } from '../../../test/util';
+import { GlobalConfig } from '../../config/global';
 import { simpleGitConfig } from './config';
 
+jest.mock('../../config/global');
+const globalConfig = mocked(GlobalConfig);
+
 describe('util/git/config', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
```suggestion
    jest.resetAllMocks();
    GlobalConfig.reset();
```

Reply:
Removed the reset mocks since I removed the mocks